rmo/notification/v1/rmo_ntf_v1.0.0.yaml

```diff
@@ -0,0 +1,268 @@
+openapi: 3.0.0
+x-stoplight:
+  id: 56d39d62070e0
+info:
+  version: 1.0.0-Beta-1
+  title: DCSA OpenAPI specification for Operational Reefer Notifications
+  description: |
+    API specification issued by DCSA.org.
+
+    Operational Reefer Monitoring Notifications for [DCSA OpenAPI specification for Operational Reefer Monitoring](https://app.swaggerhub.com/apis/dcsaorg/DCSA_RMO/1.0.0-Beta-1) is a lightweight notification based on [CloudEvents](https://cloudevents.io/). The `POST` endPoint of the consumer is called whenever a `Reefer` that is being subscribed to changes state or is updated.
```

Comment:
Does the Reefer change state? In Booking - the Booking actually changes state from `RECEIVED` to `PENDING UPDATE` - that is why I have used that wording... It might also be correct for Reefer?

Reply:
I've modified and will clarify with Alex, your correct to question as when I read back maybe state isn't the right word, in the Reefer Monitoring API the consumer will subscribe to agreed notification types with their provider it maybe an alarm being activated on the reefer, or a geofence being triggered.  I've re-worded and submitting change.